rust_snuba/src/processors/spans.rs

```diff
@@ -270,6 +261,7 @@ impl TryFrom<FromSpanMessage> for Span {
             trace_id: from.trace_id,
             transaction_id: from.event_id,
             transaction_op,
+            user: from.sentry_tags.user.unwrap_or_default(),
```

Comment:
Setting a `user` column value was somehow forgotten despite the column being there.